src/Utils/ArrayDiff.php

```diff
@@ -48,4 +50,40 @@ public function diff($oldData, $newData)
 
         return $diff;
     }
+
+    /**
+     * Compare the type and the property values of two objects.
+     * Return true if they are the same, false otherwise.
+     * If the type is the same and all properties are the same, this will return true, even if they are not the same instance.
+     *
+     * @see https://www.php.net/manual/en/language.oop5.object-comparison.php.
```

Comment:
```suggestion
     * @see https://www.php.net/manual/en/language.oop5.object-comparison.php
```
(The link with the dot is a 404 for me)

Reply:
Fixed as part of my previous commit